lib/internal/process.js

```diff
@@ -84,8 +84,7 @@ function setup_hrtime() {
         const needsBorrow = nsec < 0;
         return [needsBorrow ? sec - 1 : sec, needsBorrow ? nsec + 1e9 : nsec];
       }
-      throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-                                 'process.hrtime()', 'Array');
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'time', 'Array');
```

Comment:
This comes with a problem. After adding the actual argument, the error message looks like `'The "time" argument must be of type Array. Received type Array'` for arrays with an invalid size. The correct message would be something like `'The "time" array must have a length of 2. Received 3'`. Any ideas for a solution? Maybe a new error code `ERR_INVALID_ARRAY_LENGTH`?